lib/components/T3CeFeloginLogin/T3CeFeloginLogin.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+  <div class="fe-login">
+    <T3Form ref="form" :elements="elements" @submit="onSubmit" />
+  </div>
+</template>
+<script>
+import T3Form from '../../components/T3Form/T3Form'
+import baseCe from '../../mixins/component/baseCe'
+
+export default {
+  name: 'T3CeFeloginLogin',
+  components: {
+    T3Form
+  },
+  extends: baseCe,
+  props: {
+    data: {
+      type: Object,
+      required: true
+    }
+  },
+  computed: {
+    elements () {
+      return this.data.form.fields.pages
+    }
+  },
+  methods: {
+    onSubmit ({ validator, form, formRef }) {
+      const data = new FormData(formRef)
+      this.$typo3.api.$http
+        .$post(this.data.form.action, data)
+        .then((data) => {
+          alert(data)
```

Comment:
we have some knowledge what should we do after login - there is redurect url field which we can use to action on success.
<img width="971" alt="Zrzut ekranu 2021-12-1 o 16 01 43" src="https://user-images.githubusercontent.com/7935392/144258640-9b78a096-1f76-43b5-854f-9ecfd5b31673.png">
but we should reload `initialData` before redirect to get user data.